xbmc/addons/gui/skin/SkinTimer.cpp

```diff
@@ -65,6 +65,11 @@ bool CSkinTimer::VerifyStopCondition() const
   return m_stopCondition && m_stopCondition->Get(INFO::DEFAULT_CONTEXT);
 }
 
+std::string CSkinTimer::GetName() const
```

Comment:
why not return `const std::string&` ?

Reply:
some argue that if you return a const ref to a string you pretty much limit any consumer of the API. I do agree, but in this specific case since the name is the unique identifier of the timer and cannot be changed in any possible way (it has to be parsed from the timers.xml file) it's fine to return it as const ref. Changed